### DISCUSSION TITLE
Getting Started with PrimeVue?

### DISCUSSION THREAD

**Discussion by pnkonstant** [primefaces/community#2746] [PrimeVue]
Hello,

I am trying to get started with PrimeVue.  I did purchase the product and received the .zip file.  Do I run 'npm install <primevue.zip file>?

Thank you
Peter

**Comment by j0rgedev** [comment]
Hello @PNKonstant. Do you mean that you purchased a PrimeVue Template?

**Reply by pnkonstant** [reply]
Yes, I did.  I bought the developer license and received a .zip file.  I assume I don’t use the typical call “npm install primevue” but need to reference the .zip file

**Reply by j0rgedev** [reply]
I haven't bought any prime vue template but I guess the one you purchased, you should unzip it and install dependencies with **npm install** as you said

**Comment by pnkonstant** [comment]
Yes, I did.  I bought the developer license and received a .zip file.  I assume I don’t use the typical call “npm install primevue” but need to reference the .zip file.


Sent from my iPhone

On Sep 28, 2024, at 8:07 PM, Jorge A. ***@***.***> wrote:

﻿

Hello @PNKonstant<https://github.com/PNKonstant>. Do you mean that you purchased a PrimeVue Template?
